distribution/ddf-common/src/main/resources/etc/org.ops4j.pax.logging.cfg

```diff
@@ -17,113 +17,207 @@
 #
 ################################################################################
 
-# In order to switch to Log4j2 and be able to take advantage of the newer appenders,
-# such as the Failover appender, uncomment the following line and remove all the
-# configuration beneath it.
-#
-# This change can be made permanent (or done in another way, depending upon
-# the implementation choices made) when we have upgraded to Karaf 4.1.0
-# https://codice.atlassian.net/browse/DDF-2132
+# In order to switch to the xml-style configuration for Log4j2 in ${karaf.etc}/log4j2.config.xml, 
+# uncomment the following line and remove all the configuration beneath it. This change has not been 
+# made default because the xml-style configuration causes log entries to be missing from the itests.
 #
 # org.ops4j.pax.logging.log4j2.config.file = ${karaf.etc}/log4j2.config.xml
 
+# Common pattern layout for appenders
+log4j2.pattern = %-5.5p | %-16.16t | %-32.32c{1} | %-32.32C %4L | %X{bundle.id} - %X{bundle.name} - %X{bundle.version} | %m%n
```

Comment:
@vinamartin is right. No, the location information is only an issue for logs in the console, like you see in `log:tail`, and the format for that is defined in `org.apache.karaf.log.cfg`. The log entries in `etc/log` will contain location details by default.